### PR TITLE
Fixed 19 build time warnings

### PR DESCRIPTION
**Pull request by hiteshsharma**
See all the warnings here
https://scans.gradle.com/s/n3ap5h46hu6ne/console-log#L84

### CODE REVIEW

File: subprojects/parseq/src/main/java/com/linkedin/parseq/Engine.java [hunk 1]
```diff
@@ -58,7 +58,7 @@
   private static final int DEFUALT_MAX_RELATIONSHIPS_PER_TRACE = 65536;
 
   public static final String MAX_CONCURRENT_PLANS = "_MaxConcurrentPlans_";
-  private static final int DEFUALT_MAX_CONCURRENT_PLANS = Integer.MAX_VALUE;
+  private static final int DEFAULT_MAX_CONCURRENT_PLANS = Integer.MAX_VALUE;
```
Comment (mchen07) [inline subprojects/parseq/src/main/java/com/linkedin/parseq/Engine.java]: Is this already in another PR?

File: subprojects/parseq/src/main/java/com/linkedin/parseq/internal/PlanDeactivationListener.java [hunk 2]
```diff
@@ -5,8 +5,6 @@
 /**
  * This listener interface allows receiving notifications about plan deactivation.
  * It can be used for monitoring and to optimize task implementation e.g. to implement batching.
- * See {@link EngineBuilder#setPlanActivityListener(PlanDeactivationListener)}
```
Comment (mchen07) [inline subprojects/parseq/src/main/java/com/linkedin/parseq/internal/PlanDeactivationListener.java]: Why do you remove this? This method is there in `EngineBuilder`.

File: subprojects/parseq/src/main/java/com/linkedin/parseq/Engine.java (hunk 1 above)
Comment (hiteshsharma) [reply subprojects/parseq/src/main/java/com/linkedin/parseq/Engine.java]: Deleted that PR. This should be the only one with this change now

File: subprojects/parseq/src/main/java/com/linkedin/parseq/internal/PlanDeactivationListener.java (hunk 2 above)
Comment (hiteshsharma) [reply subprojects/parseq/src/main/java/com/linkedin/parseq/internal/PlanDeactivationListener.java]: This method was replaced by `setPlanDeactivationListener` in https://github.com/linkedin/parseq/commit/bfcc5ffb2680fc36e9a7bf4f139f8df479bb2b4d but this comment never got changed. I wasn't able to find this method that's why I removed it but let me replace it with the new one

**Pull request merged**